src/lib/components/documents/PDFPage.svelte

```diff
@@ -189,6 +212,38 @@ Selectable text can be rendered in one of two ways:
         <!-- pdfjs.renderTextLayer will fill this in -->
       </div>
     {/if}
+    {#if notes}
+      {#await pdf then pdf}
+        <div class="notes">
+          {#each notes as note}
+            <a
+              class="note"
+              href={noteHashUrl(note)}
+              title={note.title}
+              style:top="{note.y1 * 100}%"
+              on:click={(e) => openNote(e, note)}
+            >
+              <NoteTab access={note.access} />
+              {#if note.id === $activeNote?.id}
+                <button
+                  class="close"
+                  on:click|preventDefault|stopPropagation={closeNote}
+                >
+                  <XCircleFill16 />
+                </button>
+              {/if}
+            </a>
+
+            <Note
+              {note}
+              focused={note.id === $activeNote?.id}
+              {pdf}
+              scale={numericScale}
+            />
```

Comment:
I'm using the focused state to render the notes view (`Notes.svelte`). This will probably also become the note embed eventually. In both those cases, we won't have an `$activeNote` store.

Reply:
I think that's included in what I'm saying—not clear why focused is necessary if it's permanently set to `true` in the Notes view and the embed. Seems like we could just render the note without it and simplify the component's API.

Reply:
Ah, I get what you're saying. So we'd move the note link up into the `PDFPage` component. That does make sense.

Reply:
This is done. There's now a `NoteLink.svelte` component, so that's cleaner now.